addons/test/test-misra.py

```diff
@@ -137,3 +137,10 @@ def test_rules_cppcheck_severity(checker):
     assert("(error)" not in captured)
     assert("(warning)" not in captured)
     assert("(style)" in captured)
+
+
+def test_misra_18_7(checker):
```

Comment:
This is redundant .. Travis executes this command:

     ~/cppcheck/cppcheck --dump misra-test.c && python ../misra.py -verify misra-test.c.dump

And this command will verify that the addon outputs an error report on every marked line and nowhere else.